packages/generator/src/function-import/function.ts

```diff
@@ -51,13 +51,37 @@ function getFunctionImportStatements(
       }, 'const params = {\n') + '\n}'
     : '{}';
 
-  const returnStatement = `return new FunctionImportRequestBuilder('${
-    functionImport.httpMethod
-  }', '${service.servicePath}', '${
-    functionImport.originalName
-  }', (data) => ${responseTransformerFunctionName(
-    functionImport.returnType
-  )}(data, ${functionImport.returnType.builderFunction}), params);`;
+  const parameters =
+    service.oDataVersion === 'v2'
```

Comment:
Wasn't there a function for this, like `isODataV2`?

Reply:
I only found the one action on edmx metadata. I will have a look.

Reply:
I only see the isV2Metadata which is  not right here.